plugins/send2ebook.koplugin/main.lua

```diff
@@ -227,6 +227,7 @@ function Send2Ebook:onCloseDocument()
         logger.dbg("Send2Ebook: download_dir_path:", download_dir_path)
         logger.dbg("Send2Ebook: removing Send2Ebook file from history.")
         ReadHistory:removeItemByPath(document_full_path)
+        self.ui:setLastDirForFileBrowser(document_full_path)
```

Comment:
Shouldn't it be `download_dir_path` (a dir, instead of document_full_path which is a file) ?
The same below for walalbag?